pkg/integrations/v2/ebpf/integration_stub.go

```diff
@@ -0,0 +1,25 @@
+//go:build !linux || !ebpf_enabled
+// +build !linux !ebpf_enabled
+
+package ebpf
+
+import (
+	"github.com/grafana/agent/pkg/integrations/v2"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+)
+
+func init() {
+	integrations.Register(&config{}, integrations.TypeSingleton)
+}
+
+type config struct{}
```

Comment:
I think we need the config defined here also *or better yet in a shared file*. Unmarshal strict will fail if the fields cant find a home on a nonlinux platform.

Reply:
You're right! I opted to re-define the only yaml field we need here, instead of having a common file, as the stub config is slimmer.

It just felt better in the moment, but I'm not opposed to changing this.